content-src/experiments/universal-search.yaml

```diff
@@ -94,74 +94,111 @@ modified: '2016-07-20T15:42:28.873321Z'
+  high CTR for showing only three result types and overall user sentiment was quite positive.</p>
+  <p><strong>Results from telemetry data</strong></p>
+  <p>The data showed a final average CTR of 7 percent, and indicated a clear preference 
+  for top-level domain results. A breakdown of clicks by result type showed that 
+  Wikipedia pages were shown most and clicked least. Top-level domains were the 
+  most consistent performer: our better relevancy on top-level domains (even 
+  after de-duping) probably indicated we were short-cutting users to sites 
+  they already knew they wanted to visit. Movie Cards had the least amount of test data, 
+  but outperformed Wikipedia pages.</p>
+  <p>Search Results Displayed and Clicked</p>
   <figure>
-  <img src="/static/images/experiments/universal-search/graduation/results_1.png" />
-  <figcaption>Percentage of featured cards shown and taken during the experiment.</figcaption>
+  <img src="/static/images/experiments/universal-search/graduation/graph1.png" />
+  <figcaption>
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph1_key1.png" /> Displayed<br />
+  <img style="width:36px;" src="/static/images/experiments/universal-search/graduation/graph1_key2.png" /> Clicked
+  </figcaption>
   </figure>
-  <p>For the first two months, we showed users a result approximately 20% of the
-  time they typed in the Awesome Bar, and users clicked on a displayed result
-  about 10% of the time. In mid-July, we made a significant change to exclude
-  showing a search result if the destination page was already in the user’s
-  history. We expected this to dramatically reduce the frequency of results
-  shown, and it did. Results displayed fell approximately 2/3, from 20% to 7%.
-  What we didn’t expect was that the percentage of results clicked would also
-  fall: we showed fewer results, and our click-through rate was worse.
-  Intuitively this made sense. The duplicates we removed would include many
-  sites that users revisit habitually, so our previously higher click-through
-  rate included many clicks stolen from history navigation.</p>
-  <img src="/static/images/experiments/universal-search/graduation/results_2.png" />
-  <figcaption>Card types shown and selected. We added movie cards in early October.</figcaption>
+  <p><figcaption>User data showing percentage of time recommendations were displayed and 
+  percentage of time participants clicked on recommendations.</figcaption></p>
+  <p>Key events to note in the graph<br />
+  <ul><li>May to July -  Top-level domain and Wikipedia results displayed </li>
+  <li>Mid-July to Mid-August - No data collected during telemetry system upgrade</li>
+  <li>Mid-August to September - Top-level domain (de-duped) and Wikipedia results displayed</li>
+  <li>October to November - Rich movie card, top-level domain (de-duped) and Wikipedia results displayed</li></ul></p>
```

Comment:
Another https://validator.w3.org/nu/#textarea error:

> Error: No `p` element in scope but a `p` end tag seen.

I think that maybe the `</p>` should be closed on L155 above (instead of the `<br />`) and the `<ul>` is just standalone.